src/Header/Account/Account.test.jsx

```diff
@@ -149,4 +155,103 @@ describe("Account", () => {
 		).toBe(false);
 		expect(wrapper.find("#my-account").props()["aria-hidden"]).toBe(true);
 	});
+
+	describe("tracking", () => {
+		beforeEach(() => {
+			window.dataLayer = [];
+		});
+
+		afterAll(() => {
+			// Cleanup
+			delete window.dataLayer;
+		});
+
+		const trackingTest = (accountsData, linkText, eventLabel, href) => {
+			const wrapper = shallow(
+				<Account
+					isLoggedIn={accountsData != null}
+					accountsData={accountsData}
+				/>
+			);
+
+			const preventDefault = jest.fn();
+
+			wrapper.find(`a[children="${linkText}"]`).simulate("click", {
+				preventDefault: preventDefault,
+				currentTarget: {
+					getAttribute: () => href
+				}
+			});
+
+			expect(window.dataLayer).toEqual([
+				{
+					event: eventName,
+					eventCategory: defaultEventCategory,
+					eventAction: headerClickEventAction,
+					eventLabel: eventLabel,
+					eventCallback: expect.any(Function),
+					eventTimeout: eventTimeout
+				}
+			]);
+
+			expect(preventDefault).toHaveBeenCalled();
+
+			window.dataLayer[0].eventCallback();
+			expect(window.location.href).toEqual(href);
+		};
+
+		it("should not send dataLayer event or prevent default for admin link click", () => {
+			const wrapper = shallow(
+				<Account
+					isLoggedIn={true}
+					accountsData={{
+						display_name: "Joe Bloggs",
+						links: {
+							Admin: "https://accounts.nice.org.uk/admin"
+						}
+					}}
+				/>
+			);
+
+			const preventDefault = jest.fn();
+
+			wrapper.find(`a[children="Admin"]`).simulate("click", {
+				preventDefault: preventDefault,
+				currentTarget: {
+					getAttribute: () => "https://accounts.nice.org.uk/admin"
+				}
+			});
+
+			expect(preventDefault).not.toHaveBeenCalled();
+
+			expect(window.dataLayer).toHaveLength(0);
+		});
```

Comment:
I feel there is a bit of duplication between this and trackingTest(). Its possible to consolidate them  however it probably decreases clarity as there are distinct differences between the two.